README.md

```diff
@@ -228,6 +228,33 @@ You're going to want to stick all of this into your `package.yaml` file.
     - TypeFamilies
 ```
 
+## Building with Nix
+
+The project provides basic nix config for building in development.
+
+To build the main library or plugin:
+
+```bash
+nix-build -A polysemy
+nix-build -A polysemy-plugin
+```
+
+To inspect a dependency:
+
+```bash
+nix repl
+
+> p = import ./.
+> p.unagi-chan
```

Comment:
This is relevant to the project, because we have `unagi-chan` as a dependency - is this correct?

Reply:
yeah I just chose one at random to showcase that the nix derivation contains the whole build